pkg/kube/config_getter.go

```diff
@@ -7,6 +7,11 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+type OIDCPrefixes struct {
```

Comment:
This is exported, so should have a doc-comment.

But, it also feels like these are not `OIDCPrefixes` at this point, yes, we might use these to match against groups that come from OIDC, but these are not related to OIDC?

Is there a better name?

Reply:
Makes sense to me 🙂 Is `UserPrefixes` suitable?